reference/scalability_guide/more_than_100k_products_to_export.rst

```diff
@@ -1,16 +1,16 @@
 More than 100k products to export?
 ----------------------------------
 
-We had use cases where customers would export 270k products and had issues with the memory usage.
+In the past, we encountered use cases where partners would export 270k products and experienced issues with the memory usage.
 
-Most of the massive operations of the PIM like imports and exports are bulked by page of lines (or objects) to avoid too large memory usage.
+Most of PIM's massive operations like imports and exports are #bulked by page of lines (or objects)# to avoid too large memory usage. #bulked by page of lines ?""
```

Comment:
we process data page by page, for instance, 100 products per 100 products, or 100 rows per 100 rows to avoid to load everything in memory